env.sh

```diff
@@ -0,0 +1,26 @@
+alias forge-fmt="cargo r --quiet -p forge-fmt --"
+forge-fmt-cmp() {
```

Comment:
@0xrusowsky nit: did we intend for this to be pushed to `master`, looks like a dev leftover